docs/HyperIndex/overview.md

```diff
@@ -5,19 +5,24 @@ sidebar_label: Overview
 slug: /overview
 ---
 
-NOTE: v2 is currently in rc phase (release candidate) and is not yet stable. Please refer to the [v1 documentation](https://docs.envio.dev/docs/HyperIndex) for the stable version. Here is the [migration guide](./migration-guide-v1-v2).
+HyperIndex is a real-time indexer built specifically for EVM-compatible blockchains (+Fuel), providing developers with a seamless and efficient indexing solution. Designed to optimize the user experience, HyperIndex offers automatic code generation and flexible language support.
 
-Envio (now referred to as HyperIndex) is a real-time indexer built specifically for EVM-compatible blockchains, providing developers with a seamless and efficient indexing solution. Designed to optimize the user experience, Envio offers automatic code generation and flexible language support.
-Indexers on Envio can be written in JavaScript, TypeScript or ReScript.
+## Key Features
 
-Indexing blockchain data becomes effortless with Envio, offering fast syncing speed and simplified setup steps. Detailed logging and error messaging are provided for effective troubleshooting and debugging.
+- Fast syncing speed and simplified setup
+- Automatic code generation
+- Support for JavaScript, TypeScript, and ReScript
+- Detailed logging and error messaging for effective troubleshooting
+- Ability to aggregate and index data from multiple chains
```

Comment:
Join on-chain and off-chain data for enriched application API?

Reply:
Yeah I think an improvement to add in future. I want to make a Why Envio section where we do this